src/rpc/misc.cpp

```diff
@@ -212,85 +212,57 @@ class CWallet;
 /**
  * Used by addmultisigaddress / createmultisig:
  */
-CScript _createmultisig_redeemScript(CWallet * const pwallet, const UniValue& params)
+extern void _createmultisig_addr_to_pubkey(CWallet* const pwallet, CPubKey& pubkey_out, const std::string& addr_in);
```

Comment:
nit: for functions which are not put in headers but used expernally, can you put a comment where the function is defined noting that it is duplicated elsewhere so people wishing to change its API are aware.

Reply:
Renamed to `AddrToPubKey` and the extern declaration was moved to `rpc/util.h`. Also added a comment with where it is defined.